packages/grafana-ui/src/components/index.ts

```diff
@@ -2,20 +2,23 @@ export { DeleteButton } from './DeleteButton/DeleteButton';
 export { Tooltip } from './Tooltip/Tooltip';
 export { Portal } from './Portal/Portal';
 export { CustomScrollbar } from './CustomScrollbar/CustomScrollbar';
-export { Label } from './Label/Label';
 
 // Select
 export { Select, AsyncSelect, SelectOptionItem } from './Select/Select';
 export { IndicatorsContainer } from './Select/IndicatorsContainer';
 export { NoOptionsMessage } from './Select/NoOptionsMessage';
 export { default as resetSelectStyles } from './Select/resetSelectStyles';
 
+// Forms
+export { FormLabel } from './FormLabel/FormLabel';
+export { FormField } from './FormField/FormField';
+export { Label } from './Label/Label';
```

Comment:
this component seems like a dupe of FormLabel?

Reply:
No? They have different props, and Label has a Tooltip?

Reply:
but can't we add that too FormLabel? feels like we only need one FormLabel component when they are this similar

Reply:
Sure